modules/pxp-module-puppet

```diff
@@ -154,7 +156,7 @@ def metadata()
           },
           :required => [:env, :flags]
         },
-        :output => {
+        :results => {
```

Comment:
Well this is changing output (from the old way of running a module?) to results (from the new way I assume) so I'd expect this change to be part of the 'port to the new api' change, rather than the 'add this new behaviour' change.